qa-tests-backend/src/test/groovy/SummaryTest.groovy

```diff
@@ -111,11 +111,7 @@ class SummaryTest extends BaseSpecification {
     }
 
     @Tag("BAT")
-    @IgnoreIf({ System.getenv("OPENSHIFT_CI_CLUSTER_CLAIM") == "openshift-4" })
     def "Verify namespace details"() {
-        // https://issues.redhat.com/browse/ROX-6844
-        Assume.assumeFalse(ClusterService.isOpenShift4())
-
```

Comment:
You wrote, you want to enable it for OS later :)

Reply:
Yeah, I wanted to run CI one more time in ocp. I've reverted the last commit with those changes 🙂